src/App.js

```diff
@@ -1,25 +1,80 @@
-import logo from './logo.svg';
-import './App.css';
+import logo from "./logo.svg";
+import "./App.css";
+import React, { useState } from "react";
+import CoinImage from "./components/CoinImage";
+import WinMessage from "./components/WinMessage";
 
-function App() {
+// function App() {
+//   return (
+//     <div className="App">
+//       <header className="App-header">
+//         <img src={logo} className="App-logo" alt="logo" />
+//         <p>
+//           Edit <code>src/App.js</code> and save to reload.
+//         </p>
+//         <a
+//           className="App-link"
+//           href="https://reactjs.org"
+//           target="_blank"
+//           rel="noopener noreferrer"
+//         >
+//           Learn React
+//         </a>
+//       </header>
+//     </div>
+//   );
+// }
+
+// export default App;
+
+export default function App() {
+  const [heads, setHeads] = useState(0);
+  const [tails, setTails] = useState(0);
+  const [coin, setCoin] = useState(null); // coin is the value associated w H or T coin // setCoin is the function that changes the value of coin
+
+  function Reset() {
```

Comment:
🔵 By convention, functions are usually camelCase unless they are a component. Following that, this should be `reset` rather than `Reset`.